templates/web/pages/reset_password/reset_password.tt.html

```diff
@@ -5,7 +5,7 @@
     [% INCLUDE 'web/common/includes/error_list.tt.html' %]
     <form method="post" action="/cgi/reset_password.pl" enctype="multipart/form-data">
         [% IF type == 'send_email' %]
-            <label>
+            <label for="username or email">
```

Comment:
I don't think this is a correct id

Reply:
> 

Yes @VaiTon I was unsure of this. I think it should be `userid_or_email`

Reply:
I agree. You should add it also to the input under this line!